nipype/interfaces/dcm2nii.py

```diff
@@ -485,6 +485,11 @@ def _parse_files(self, filenames):
                     mvecs.append(fl)
                 elif fl.endswith(".json") or fl.endswith(".txt"):
                     bids.append(fl)
+
+        # in siemens mosaic conversion nipype misread dcm2niix output and generate a duplicate list of results
+        # next line remove duplicates from output files array
+        outfiles = [*set(outfiles)]
```

Comment:
If no test is feasible, fair enough. This will at least preserve the ordering, in case anything depends on it:

```suggestion
        outfiles = list(dict.fromkeys(outfiles))
```